internal/validate/validate.go

```diff
@@ -189,7 +189,7 @@ func DBUsername(val interface{}) error {
 	return fmt.Errorf("%w: %s", ErrInvalidDBUsername, name)
 }
 
-var ErrWeakPassword = errors.New("the password provided is too common")
+var ErrWeakPassword = errors.New("the password provided is too pointers")
```

Comment:
I am not sure what this mean

Reply:
Sorry, this is a mistake made when renaming a package

Reply:
Fixed